03/app.test.js

```diff
@@ -1 +1,28 @@
-import randomNumber from './app';
\ No newline at end of file
+import RanNum from './app';
+
+/*test('is randomNumber a function', () => {
+    const ranNum = new RanNum(1, 2);
+    expect(ranNum.randomNumber).toBe('function')
+
+})  */
+// tu po przebudowaniu funckji w app.js i dodaniu return pojawil sie problem (Expected: "function"
+//    Received: [Function randomNumber])
+//
+
+
+
```

Comment:
Tutaj spokojnie mozna dołożyć dodatkowe testy:
- czy jak podasz przez parametr inne wartości jak liczby czy będzie błąd (dla pierwszego i drugiego parametru)
- czy jak dodasz w drugim parametrze wartość większą czy będzie zwrócony błąd
- czy jak wygenerujesz losową liczbę to będzie ona w odpowiednim przedziale